js/color-toggle.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Originally from: https://mdbootstrap.com/docs/standard/content-styles/theme/
+ * with modifications to support Bootstrap
+ */
+
+const themeStitcher = document.getElementById("themingSwitcher");
```

Comment:
Possible typo?

Reply:
It was a typo, fixed in [8171400](https://github.com/AndrewPaglusch/FlashPaper/pull/159/commits/8171400912bc3ba6b3135d2e04231c90ee80b1bb)